prog/dftb+/lib_poisson/bulkpot.F90

```diff
@@ -342,7 +335,7 @@ Subroutine readbulk_pot(phi_bulk, iErr)
     if (a.ne.phi_bulk(m)%iparm(14) .or. &
       b.ne.phi_bulk(m)%iparm(15) .or. &
       c.ne.phi_bulk(m)%iparm(16)) then
-      if(id0) write(*,*) 'Warning: incompatible BulkPot: will be recomputed'   
+      call warning('incompatible BulkPot: will be recomputed')
```

Comment:
Is this automatically protected for the MPI master?

Reply:
Yes, the stdOut unit is directed to /dev/null on all but the master

Reply:
Sounds hacky, but okay.